project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import Keys._
 
 object Dependencies {
   val Versions = Seq(
-    crossScalaVersions := Seq("2.10.5", "2.11.6"),
+    crossScalaVersions := Seq("2.11.8"),
```

Comment:
I think you can leave the crossScalaVersion from 2.11.0 to 2.11.8 and only take the latest one using `crossScalaVersions.value.reverse.head`